### PR TITLE
Add banks in Hong Kong

### PR DESCRIPTION
**Pull request by tommylung**
Added
- 中信銀行（國際） China CITIC Bank International
- 中國工商銀行（亞洲） Industrial and Commercial Bank of China (Asia)
- 中國建設銀行（亞洲） China Construction Bank (Asia)
- 交通銀行 Bank of Communications
- 花旗銀行 Citibank
- 南洋商業銀行 Nanyang Commercial Bank
- 華僑永亨銀行 OCBC Wing Hang Bank
- 盤谷銀行 Bangkok Bank

### CODE REVIEW

File: data/brands/amenity/bank.json [hunk 1]
```diff
@@ -7646,7 +7646,7 @@
         "amenity": "bank",
         "brand": "Raiffeisen Bank",
         "brand:wikidata": "Q16522506",
-        "brandwikipedia": "hu:Raiffeisen Bank (Magyarország)",
+        "brand:wikipedia": "hu:Raiffeisen Bank (Magyarország)",
```
Comment (LaoshuBaby) [inline data/brands/amenity/bank.json:7649]: Thank you for finding this mistake!

File: data/brands/amenity/bank.json [hunk 2]
```diff
@@ -13520,20 +13598,20 @@
...
+        "brand": "東亞銀行 Bank of East Asia",
         "brand:en": "Bank of East Asia",
         "brand:wikidata": "Q806679",
         "brand:wikipedia": "zh:東亞銀行",
         "brand:zh-Hans": "东亚银行",
         "brand:zh-Hant": "東亞銀行",
         "name": "東亞銀行 BEA",
-        "name:en": "Bank of East Asia",
+        "name:en": "BEA",
         "name:zh-Hans": "东亚银行",
```
Comment (LaoshuBaby) [inline data/brands/amenity/bank.json:13615]: Will `short_name=BEA` better? And it can be match.

**Review by LaoshuBaby — APPROVED** [review]

File: data/brands/amenity/bank.json (hunk 2 above)
Comment (tommylung) [reply data/brands/amenity/bank.json:13615]: Yes, we can add as `short_name`. 
Did `short_name` also have bilingual tags? Like `short_name:en` `short_name:zh` something like that.

Comment (LaoshuBaby) [reply data/brands/amenity/bank.json:13615]: > `short_name:en` `short_name:zh`

Yes, we have this kind of usage:

https://github.com/osmlab/name-suggestion-index/blob/cb5996bbb9f84c71abfc849e4c73d9a5a3dc1211/data/brands/shop/supermarket.json#L9198-L9203

**Pull request merged**